Modules/_sysconfig.c

```diff
@@ -55,6 +55,23 @@ _sysconfig_config_vars_impl(PyObject *module)
         Py_DECREF(config);
         return NULL;
     }
+
+    // On Unix, the `ABIFLAGS` key is defined via a different logic.
+    //
+    // Emulate `sys.abiflags` value on Unix for Windows. ABIFLAGS here is only
+    // an emulated value. It is not present during build on Windows.
+    if (add_string_value(config, "ABIFLAGS",
+#  ifdef Py_GIL_DISABLED
+                         "t"
+#  endif
+#  ifdef _DEBUG
+                         "d"
+#  endif
+                         "")
+        < 0) {
+        Py_DECREF(config);
+        return NULL;
+    }
```

Comment:
Instead of setting `ABIFLAGS` here, I'd prefer to add `Py_DEBUG` and then construct `ABIFLAGS` in `sysconfig._init_non_posix`.

Reply:
Agree with adding `Py_DEBUG`, but if anything we should construct `ABIFLAGS` further back in the build (e.g. in a `.props` file) and store that value here.

We don't need or use it, though, so I don't see an issue with constructing it here. But I'd rather have build variables be closer to the build rather than further away.

Reply:
I do not have a strong preference for whether the code should live in Python or C. Both are fine for me.

Reply:
> I do not have a strong preference for whether the code should live in Python or C. Both are fine for me.

I move the definition from C to Python in the latest commit. I can either keep it or revert it if we reach a consensus.